npm/src/ee/federated-saml/sso.ts

```diff
@@ -71,6 +71,7 @@ export class SSO {
         product: app.product,
         idp_hint,
         authFlow: 'saml',
+        samlFedAppId: app.id,
```

Comment:
SAML federation app ID is required to fetch the branding on the IDP selection page.